cli/cmd/plugin/conformance/e2e-test/e2e_test.go

```diff
@@ -0,0 +1,117 @@
+//go:build e2e
+
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_test
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"math/big"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+var tceRepoPath string
+var clusterName string
+var conformanceDirPath string
+var unmanagedClusterDirPath string
+var conformanceResultFilename string
+
+func initializeVariable() {
+	// Variables used across functions
+	tceRepoPath = os.Getenv("ROOT_DIR")
+	conformanceDirPath = tceRepoPath + "/cli/cmd/plugin/conformance"
+	unmanagedClusterDirPath = tceRepoPath + "/cli/cmd/plugin/unmanaged-cluster"
+	rand, _ := rand.Int(rand.Reader, big.NewInt(1000))
+	clusterName = "uc" + rand.String() + "conformance-test"
+}
+
+func TestSetup(t *testing.T) {
+	initializeVariable()
+
+	err := os.Chdir(unmanagedClusterDirPath)
+	if err != nil {
+		t.Errorf("error while changing directory to unmanged-cluster: %v", err)
+	}
+	// Creating Unmanaged cluster
+	_, err = cliRunner("go", nil, "run", ".", "create", clusterName)
```

Comment:
This should use the `--tty-disable` flag to prevent the serialized emojis and stylized logging. The output will look much nicer

Reply:
Hey @jpmcb I add this flag but due to the way our cliRunner() function is printing the log we are not achieving the good way of Printing logs. I am investigating into this  will change once whenever found the root cause. For now I am adding a future task as comment in the code.
Thanks!